units/migrate_volume.go

```diff
@@ -231,7 +236,20 @@ func (j *volumeMigrationJob) populateIfUnset() error {
 	}
 
 	if j.InitialHostID == "" {
-		j.InitialHostID = j.volume.Host
+		// If volume was attached to terminated host, query this host by its home volume field.
```

Comment:
Phrasing question - is this implying that we attached the volume to a host that got terminated, or that the initial host ended up terminated (e.g. because it expired) while it still had its home volume set?

Reply:
The latter (this is currently the approach for users to transfer a home volume to a new host — terminate the workstation and spin up a new one with the volume attached). Will make this more clear!